modules/nvidia_plugin/CMakeLists.txt

```diff
@@ -92,14 +92,23 @@ endif()
 find_library(CUDNN_PATH
              NAMES cudnn
              ${HAS_REQUIRED}
-             HINTS "$ENV{CUDNN_PATH}"  "${CUDA_TOOLKIT_ROOT_DIR}" "${CUDA_PATH}" "$ENV{CUDA_PATH}"
+             HINTS "$ENV{CUDNN_PATH}" "${CUDA_TOOLKIT_ROOT_DIR}" "${CUDA_PATH}" "$ENV{CUDA_PATH}"
              PATH_SUFFIXES nvidia/current lib64 lib/x64 lib)
 
+if((${CUDAToolkit_VERSION_MAJOR} LESS_EQUAL 10))
```

Comment:
```suggestion
if((CUDAToolkit_VERSION_MAJOR LESS_EQUAL 10))
```

Reply:
For consistency I would prefer to use as it is, because it is already used in such way, see https://github.com/openvinotoolkit/openvino_contrib/pull/739/files#diff-05d66e70f2101a3454f9f9c18c2cad8de49c06c81fefa52d3280b72a02ae49daR133

Reply:
I propose to clean-up in all places. 

Having unnecessary symbols in expression does not improve readability 😃

Reply:
Done